nack/receive_log_test.go

```diff
@@ -0,0 +1,134 @@
+package nack
+
+import (
+	"reflect"
+	"testing"
+)
+
+func TestReceivedBuffer(t *testing.T) {
+	for _, start := range []uint16{0, 1, 127, 128, 129, 511, 512, 513, 32767, 32768, 32769, 65407, 65408, 65409, 65534, 65535} {
+		start := start
+
```

Comment:
Looks better to make this block subtest.
```suggestion
		t.Run(fmt.Sprintf("StartFrom%d", start). func (t *testing.T) {
```

Reply:
Fixed!